packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -238,15 +238,15 @@ class VsCode {
 
   // Linux:
   //   /usr/share/code/bin/code
-  //   /snap/code/current
+  //   /snap/code/current/usr/share/code
   //   /usr/share/code-insiders/bin/code-insiders
   // Linux Extensions:
   //   $HOME/.vscode/extensions
   //   $HOME/.vscode-insiders/extensions
   static List<VsCode> _installedLinux(FileSystem fileSystem, Platform platform) {
     return _findInstalled(<VsCodeInstallLocation>[
       const VsCodeInstallLocation('/usr/share/code', '.vscode'),
-      const VsCodeInstallLocation('/snap/code/current', '.vscode'),
```

Comment:
should we leave the old one, or am I being overly cautious?

Reply:
Just in case I've done more testing and checked with the latest Ubuntu 22.04.3 LTS, Ubuntu 20.04 LTS, Ubuntu 18.04 LTS, Ubuntu 16.04 LTS and Fedora Workstation 38. It's safe to remove the old path.

Reply:
sounds good to me